optaplanner-docs/src/main/asciidoc/QuickStart/CloudBalancingDomainModel/CloudBalancingDomainModel-section.adoc

```diff
@@ -8,25 +8,30 @@
 
 Using a <<domainModelingGuide,domain model>> helps determine which classes are planning entities and which of their properties are planning variables. It also helps to simplify contraints, improve performance, and increase flexibility for future needs. 
 
+To create a domain model, define all the objects that represent the input data for the problem. In this simple example, the objects are processes and computers. 
+
+A separate object in the domain model must represent a full data set of problem, which contains the input data as well as a solution. In this example, this object holds a list of computers and a list of processes. Each process is assigned to a compter; the distribution of processes between computers is the solution.
+
 . Draw a class diagram of your domain model. 
 . Normalize it to remove duplicate data.
-. Write down some sample instances for each class. 
+. Write down some _sample instances_ for each class. Sample instances are entity properties that are relevant for planning purposes.
 
 * ``Computer``: represents a computer with certain hardware and maintenance costs. 
 +
 In this example, the sample instances for the `Computer` class are: `cpuPower`, `memory`, `networkBandwidth`, `cost`. 
 * ``Process``: represents a process with a demand. Needs to be assigned to a `Computer` by Planner. 
 +
 Sample instances for ``Process`` are: `requiredCpuPower`, `requiredMemory`, and `requiredNetworkBandwidth`.
-* ``CloudBalance``: represents a problem. Contains every `Computer` and `Process` for a certain data set. 
+* ``CloudBalance``: represents the distribution of processes between computers. Contains every `Computer` and `Process` for a certain data set. 
```

Comment:
-1 CloudBalance represents both a planning problem and a planning solution (but we explain that 2th one later).

Reply:
Yes, but we need something for this list, and "represents a problem" is just not clear ("a problem" can be anything). Perhaps "represents a set of all processes and its distribution between computers"?

Reply:
Let's rollback the changes in this paragraph.

Reply:
If we just roll back we do no work at all. As I wrote, "represents a problem" is not clear. We need the right version.

Reply:
As requested in email, rolled back for this PR but will submit next version in a separate PR.